content/news/playtest-20150329.md

```diff
@@ -0,0 +1,41 @@
+
+Since the last playtest, we have continued our focus on the Dune 2000 mod, which has received a brand new sidebar. The sandworms feature has been polished, and harvester insurance has been added, so if you lose your only harvester to a sandworm, it will be replaced automatically, for free!
+
+Those players that mourned the loss of the left-click orders option will be glad to hear that is has returned, better than ever. A new contributor took it upon himself to re-implement it in a way that avoided the numerous problems the earlier implementation had.
+
+Other highlights include:
+
+   - In sell mode, a strucuture's tooltip will show the expected refund value.
+   - Spy planes in Red Alert will no longer lure your anti-air capable units across the map.
+   - Airplanes will now fly higher and not push helicopters out of their way anymore.
+   - Harvester logic has been improved to make use of all available refineries and to avoid blockages.
+   - New missions: Allies03b and Allies05a for Red Alert, and GDI05a, Nod02a and Nod02b for Tiberian Dawn.
+   - More memory usage and performance improvements.
+   - On Linux, the `.orarep` file extension will be associated with OpenRA so that replays can be started from a file manager with a simple double-click.
+
+[The changelog](http://changelog.openra.net) has the full list of fixes, changes and new features.
+
+Head over to our [download section](/download/) to find the playtest installers.
+
+A note on map compatibility: already with the last playtest a new map format was introduced which is not backward-compatible with last October's release-20141029. This means that if you have downloaded custom maps from [the Resource Center](http://resource.openra.net) or elsewhere, these maps will have been automatically updated to the new map format once you have run this or the previous playtest. If you later go back to the release version, it will not be able to parse the new map format and crash. You can avoid the problem by either deleting the contents of your `map` folder, or by making backups of the `map` folder and restoring that when you switch back to release-20141029. We should have mentioned that in the news post for the last playtest already, and apologize for any inconvenience that omission caused.
+
+<div style="text-align:center" markdown="1">
+![Random Countries](/images/news/20150329-ra-randomcountries-dropdown.png)
+
+It is now possible to select a random country from among each side in Red Alert, Allies or Soviets.
+
+![Refund value in tooltip](/images/news/20150329-cnc-refund-tooltip.png)
+
+Tooltips now show the refund value when sell mode is activated.
+
+![Dune 2000 sidebar](/images/news/20150329-d2k-sidebar.png)
+
+The new Dune 2000 sidebar.
+
+![Tiberian Sun Snow Preview](/images/news/20150329-ts-snow-preview.png)
+
+Behind the scenes, the work on the Tiberian Sun mod reached another milestone.
```

Comment:
Could we elaborate on this by adding "by completing support for the original game's tilesets"?